modules/local/add_flags/main.nf

```diff
@@ -33,7 +33,7 @@ process ADD_FLAG {
     cat *_metafusion_cluster.unfiltered.cff \\
         | awk 'FNR <= 1' > header.txt
     cat *_metafusion_cluster.unfiltered.cff \\
-        | grep -iFwf $clinical_genes > tmp_clinicalgenes.txt
+        | grep -iFwf $clinical_genes | awk '\$20 != "NA" && \$22 != "NA"' > tmp_clinicalgenes.txt
```

Comment:
can you also add an `|| test $? = 1` to the end of this line? when the grep operation has zero results it seems to be erroring out.